package.json

```diff
@@ -25,14 +25,17 @@
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
     "react-modal-video": "^1.2.3",
+    "react-native-hr-component": "^2.0.3",
```

Comment:
What's this being used for?

Reply:
I originally tried to use it to make a dynamic black bar that grew or shrank with the length of the movie title, but could not get it working. Thank you for heads up. I can uninstall it.